app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -196,22 +194,22 @@ class AppVersionActivityTest {
   private fun getDateTime(dateTimeTimestamp: Long): String? {
     return oppiaDateTimeFormatter.formatDateFromDateString(
       OppiaDateTimeFormatter.DD_MMM_YYYY,
-      dateTimeTimestamp,
-      Locale.US
+      timestamp = dateTimeTimestamp,
+      locale = Locale.US
     )
   }
 
   private fun launchAppVersionActivityIntent(): ActivityScenario<AppVersionActivity> {
     val intent = AppVersionActivity.createAppVersionActivityIntent(
-      ApplicationProvider.getApplicationContext()
+      context = ApplicationProvider.getApplicationContext()
     )
     return ActivityScenario.launch(intent)
   }
 
   private fun launchAdministratorControlsActivityIntent(profileId: Int): Intent {
```

Comment:
Rename `profileId ` to `internalProfileId` and also whereever you are using it as named argument.